src/rules/no-generic-verbs.js

```diff
@@ -0,0 +1,42 @@
+let utils = require('../utils');
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			url: 'https://github.com/atomspace/eslint-plugin-naming-convention/blob/master/docs/rules/no-generic-verbs.md'
+		},
+		messages: {
+			errorMessage: `The identifier has generic verb: {{ genericVerbInVariable }}.`
+		}
+	},
+	create (context) {
+		return {
+			Identifier (node) {
+				const GENERIC_VERBS = ['set', 'get', 'array', 'check', 'validate', 'make', 'process', 'start'];
+				let variableName = node.name;
+
+				let splitedVariableName = utils.splitVariable(variableName);
+
+				let genericVerbInVariable;
+
+				for (let word of splitedVariableName) {
```

Comment:
Use `Array.prototype.find` if you want to find the first verb or `Array.prototype.filter` if you want to display to the user all violated words